src/main/kotlin/io/github/detekt/gradle/DetektKotlinCompilerPlugin.kt

```diff
@@ -36,6 +41,17 @@ class DetektKotlinCompilerPlugin : KotlinCompilerPluginSupportPlugin {
         return options
     }
 
+    private fun ConfigurableFileCollection.toDigest(): String {
```

Comment:
Could you add a testcase for this isolated function if you have time, please?

Reply:
I've added a test - please merge if everything looks OK